src/gas-hub.js

```diff
@@ -265,6 +265,11 @@ function showDiff(code, type) {
     return gasFiles.indexOf(e) < 0;
   })
   .concat(gasFiles)
+  .filter((file) => {
+    const match = file.match(/(.*?)\.(gs|html)$/);
+
```

Comment:
please remove the empty line

Reply:
Sorry, I have a habit of always putting an empty line before a return ;) Fixed.